src/loki/search.cc

```diff
@@ -22,23 +22,22 @@ template <typename T> inline T square(T v) {
   return v * v;
 }
 
-bool is_search_filter_triggered(const DirectedEdge* edge,
-                                const DynamicCost& costing,
-                                const graph_tile_ptr& tile,
-                                const Location::SearchFilter& search_filter) {
```

Comment:
no change here other than shortening names. since we call this a bunch more now an inside a bunch of `if`s i really wanted a shorter method name and to make it more closely match the other similar filter ones